src/app/content/components/PrevNextBar.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import styled, { css } from 'styled-components';
+import { ChevronLeft } from 'styled-icons/boxicons-regular/ChevronLeft';
+import { ChevronRight } from 'styled-icons/boxicons-regular/ChevronRight';
+import { decoratedLinkStyle, textRegularStyle } from '../../components/Typography';
+import theme from '../../theme';
+import { AppState } from '../../types';
+import * as select from '../selectors';
+import { Book, LinkedArchiveTreeSection, Page } from '../types';
+import ContentLink from './ContentLink';
+
+// tslint:disable-next-line:variable-name
+const LeftArrow = styled(ChevronLeft)`
+  height: 3rem;
+  width: 3rem;
+`;
+
+// tslint:disable-next-line:variable-name
+const RightArrow = styled(ChevronRight)`
+  height: 3rem;
+  width: 3rem;
+`;
+
+interface PropTypes {
+  page?: Page;
+  book?: Book;
+  prevNext?: {
+    prev?: LinkedArchiveTreeSection;
+    next?: LinkedArchiveTreeSection;
+  };
+}
+
+// tslint:disable-next-line:variable-name
+const PrevLinkElement = styled(ContentLink)`
+  ${textRegularStyle}
+  ${decoratedLinkStyle}
+  float: left;
+`;
+
+// tslint:disable-next-line:variable-name
+const NextLinkElement = styled(ContentLink)`
+  ${textRegularStyle}
+  ${decoratedLinkStyle}
+  float: right;
+`;
+
+// tslint:disable-next-line:variable-name
+const BarWrapper = styled.div`
+  height: 4rem;
+  display: flex;
+  align-items: center;
+  max-width: 57rem;
+  margin: 5rem auto 6rem auto;
```

Comment:
this bottom margin appears too large because there is also margin on top of the attribution. you either need to reduce this margin to compensate or remove the margin on the attribution.

be careful not to mess up the box-shadow on the attribution, it might need that margin for the shadow to work